MCL64/SourceCode/MCL64.ino

```diff
@@ -476,6 +477,9 @@ inline void write_byte(uint16_t local_address , uint8_t local_write_data) {
        
        wait_for_CLK_rising_edge();
        digitalWriteFast(PIN_DATAOUT_OE_n,  0x1 );   
+  }
+  if ( internal_address_check(local_address) == 0xd030) {
```

Comment:
probably here should be a check to see if I/O is enabled with current memory configuration